.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb-base"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+}
```

Comment:
Please use same config file as the one provided by microverse 
`https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json`
When you done, re-run for linting errors.

Reply:
Done! re-run for error. Found none.